### DISCUSSION TITLE
[Feature Request] Add ability to show all whitespace

### DISCUSSION THREAD

**Discussion by Megamouse** [community/community#53159] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Based on experiences shared between my co-contributors and I, it is surprisingly hard to see incorrect spacing during code reviews.
We often have to awkwardly select the changes with the mouse in order to see if it jumps by one space or by one tab.
![image](https://user-images.githubusercontent.com/23019877/232908189-c925e662-b714-4cb6-8caa-1bebed7d961a.png)

So my proposal is to add a toggle to simply show all the whitespace in a file or diff.

Example in Visual Studio with incorrect use of 4 spaces (which would be hidden to reviewers on github):
![image](https://user-images.githubusercontent.com/23019877/232907813-dd6a35a2-db6f-4490-8c01-a6f1cb984260.png)

FYI: I would also love to see an option to select tab spacing (e.g. 2 spaces or 4 spaces etc.).

**Comment by spenserblack** [comment]
This would be especially useful when editing files on the web. For example, too many times I've written an invalid workflow YAML file because I was missing 2 spaces or had 2 spaces too many. Being able to see the whitespace to help ensure the same indentation level would be really helpful. AFAIK whitespace isn't visible when editing files.
